recipe/Dockerfile

```diff
@@ -100,7 +101,7 @@ RUN if [ "x$BUILD_OMEGA_H" = xtrue ] ; then ( \
    ) fi
 
 ADD overlap-arm64.patch /var/src/
-ARG STEPS_VERSION=5.0.0_beta
+ARG STEPS_VERSION=5.0.1
```

Comment:
Since it uses `5.0.1` from the public repo, it still has the issue related to conda environments and it installs the `steps` package in the wrong directory, which makes it so that jupyter notebooks cannot `import steps`.
It can be solved (temporarily) by using `ARG STEPS_VERSION=5.0.1_conda_fixes`. The notebooks can then import steps but cannot load the bundled sundials libraries. It is probably because they are installed in `/root/.local/lib` and jupyter lab does not have the proper permissions to load them. I checked that adding `RUN chmod -R a+rwx /root` to the Dockerfile solves the issue so I think this is indeed the problem. But I am not sure how to solve it properly.